workbench-core/datasets/src/dataSetService.test.ts

```diff
@@ -222,7 +247,7 @@ describe('DataSetService', () => {
 
   describe('constructor', () => {
     it('sets a private audit and log service', () => {
-      const service = new DataSetService(audit, log, metaPlugin);
+      const service = new DataSetService(audit, log, metaPlugin, authzPlugin);
```

Comment:
Can we create a shared `service` object that gets created before the tests run so we don't have so much copy and pasting? There are examples in the `AccountsService` tests.

Reply:
@htztomic already called this out and fixed.